src/api/Services/RoleOperations.cs

```
@@ -38,47 +38,78 @@ private AuthorizationManagementClient AuthMgtClient
-	//{
-	//	var result = new Result();
+	internal RoleAssignment AssignRole(string account, string container, string role, string principalId)
+	{
+		try
+		{
+			// Get Storage Account Resource ID
+			ResourceGraphOperations rgo = new(log, TokenCredentials);
+			var accountResourceId = rgo.GetAccountResourceId(account);
 
-	//	try
-	//	{
-	//		ResourceGraphOperations rgo = new(log, TokenCredentials);
-	//		// Get Storage Account Resource ID
-	//		var accountResourceId = rgo.GetAccountResourceId(account);
+			// Create Role Assignments
+			string containerScope = $"{accountResourceId}/blobServices/default/containers/{container}";
 
-	//		// Create Role Assignments
-	//		string containerScope = $"{accountResourceId}/blobServices/default/containers/{container}";
+			// Allow user to manage ACL for container
+			var roleAssignment = AddRoleAssignment(containerScope, role, principalId);
 
-	//		// Allow user to manage ACL for container
-	//		AddRoleAssignment(containerScope, "Storage Blob Data Owner", ownerId);
+			return roleAssignment;
+		}
+		catch (Exception ex)
+		{
+			log.LogError(ex, ex.Message);
+			return null;
+		}
+	}
 
-	//		result.Success = true;
-	//	}
-	//	catch (Exception ex)
-	//	{
-	//		// TODO: Consider customizing error message
-	//		log.LogError(ex, ex.Message);
-	//		result.Message = ex.Message;
-	//	}
+	internal RoleAssignment DeleteRole(string account, string container, string rbacId)
+	{
+		try
+		{
+			// Get Storage Account Resource ID
+			ResourceGraphOperations rgo = new(log, TokenCredentials);
+			var accountResourceId = rgo.GetAccountResourceId(account);
+			var containerResourceId = $"{accountResourceId}/blobServices/default/containers/{container}";
+
+			// Get All Role Assignments for Scope
+			var roleAssignments = GetRoleAssignments(containerResourceId);
+			var authRoleAssignment = roleAssignments.FirstOrDefault(ra => ra.Id.EndsWith(rbacId));
+			if (authRoleAssignment == null)
+				return null;

```

Should we throw an exception here?